services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: str = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> MovieSession:
+
+    session = MovieSession.objects.filter(id=session_id)
+
+    if show_time:
+        session.update(show_time=show_time)
+
+    if movie_id:
+        session.update(movie=movie_id)
+
+    if cinema_hall_id:
+        session.update(cinema_hall=cinema_hall_id)
+
+    return session
```

Comment:
The `update_movie_session` function should return a `MovieSession` instance, not a `QuerySet`. Use `get` to retrieve the session and update its fields directly.